CHANGELOG.md

```diff
@@ -4,15 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [4.0.0] - 2022-02-09
```

Comment:
Bumping the major version will prevent existing users from upgrading as it implies a breaking feature change, leaving them with out critical fixes in the future (or new features that you may add later). Since this update doesn't change a single line of code that seams wrong. Probably should be `3.1.2`, or if you prefer a big number `3.2.0`

Reply:
Thank you for suggesting this :)